### ISSUE TITLE
Login (Azure) Doesn't auto sign-in when refreshing a session

### ISSUE THREAD

**Issue by lehsaclarkson** [RTWA/WebApps#21]
I've noticed that the session is not holding when logging into Webapps. It keeps logging the user out.

**Comment by RobTrehy** [comment]
This is something I had experienced, even using the local user account, in my dev environment. 
However, I put it down to not using a cached config in the dev environment - as recommended when using Laravel.

I've made a config change in commit [eac0e69](https://github.com/RTWA/WebApps/commit/eac0e6963ae16d37739b811461ca63a486413f83) which, will be included in the next WebApps release (1.0.6).
Through very quick testing, this does appear to have resolved what I believe was causing the issue.

I will leave this issue open, to gather feedback after the 1.0.6 release has been tested in your own environment.
